kernel-install/50-mkosi-initrd.install

```diff
@@ -31,7 +31,9 @@ case "$COMMAND" in
                   --cache-dir=/var/cache \
                   --output-dir="$KERNEL_INSTALL_STAGING_AREA" \
                   --extra-tree="/usr/lib/modules/${KERNEL_VERSION}:/usr/lib/modules/${KERNEL_VERSION}" \
-                  --extra-tree="/usr/lib/firmware:/usr/lib/firmware"
+                  --extra-tree="/usr/lib/firmware:/usr/lib/firmware" \
+                  --kernel-modules-exclude=".*" \
```

Comment:
I went for this because then you can just list individual modules in the config file without having to bother with paths. And it also allows other fancy stuff if needed. But yeah it is a bit of a mismatch with the other options